

package app;

message LoginReq
{
    required uint64 userid = 1;
    required string passwd = 2;
    required int32 plat = 3; // 登录平台 
}

message LoginRsp
{
    required int32 userid = 1;
    required int32 ret = 2;
}

message RegisterReq
{
    required string passwd = 1;
    required string nickname = 2;
    required int32 sex = 3;
    required uint64 birthday = 4;
    required string telno = 5;
    required string address = 6;
    required string email = 7;
}

message RegisterRsp
{
    required int32 ret = 1;
    required uint64 userid = 2;
}

message AddFriendReq
{
    required uint64 userid = 1;
}

message AddFriendRsp
{
    required int32 ret = 1;
}